validator_client/src/http_api/graffiti.rs

```diff
@@ -0,0 +1,80 @@
+use crate::validator_store::ValidatorStore;
+use bls::PublicKey;
+use slot_clock::SlotClock;
+use std::sync::Arc;
+use types::{graffiti::GraffitiString, EthSpec, Graffiti};
+
+pub async fn get_graffiti<T: 'static + SlotClock + Clone, E: EthSpec>(
```

Comment:
This function doesn't need to be `async` I think?
If we make this non-async, we can probably get rid of the `handle.block_on` from the caller too.

Reply:
whoops, yes you are correct